core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -55,9 +55,7 @@ export class ContributorDashboardPageComponent
   languageCode!: string;
   topicName!: string;
   activeTabName!: string;
-  // The following property is set to null when the
-  // user is not logged in.
```

Comment:
Just checking—Is this no longer the case?

Reply:
Oh, this was from the previous changes, I have updated it now.
Thanks